api/src/main/java/org/apache/iceberg/ManifestFile.java

```diff
@@ -41,11 +41,11 @@ public interface ManifestFile {
       "Lowest sequence number in the manifest");
   Types.NestedField SNAPSHOT_ID = optional(503, "added_snapshot_id", Types.LongType.get(),
       "Snapshot ID that added the manifest");
-  Types.NestedField ADDED_FILES_COUNT = optional(504, "added_data_files_count", Types.IntegerType.get(),
```

Comment:
Ugh, is this now used to track the number of added delete files in our delete manifests too?

Reply:
It was already mandatory for v2. But this was already the case.

Reply:
> Ugh, is this now used to track the number of added delete files in our delete manifests too?

Yeah, we decided to use the existing metadata fields for the same purpose in delete manifests. I think it makes sense -- is there a problem?